classes/command/class-duplicated-postmeta.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Detect and remove duplicate postmeta command
+ *
+ * @package P4GBKS
+ */
+
+namespace P4GBKS\Command;
+
+/**
+ * Class for detect and remove duplicate postmeta records.
+ */
+class Duplicated_Postmeta {
+	/**
+	 * Remove duplicate postmeta records
+	 *
+	 * @return int
+	 */
+	public static function remove() {
+
+		global $wpdb;
+
+		// phpcs:disable
+		$sql = "DELETE t1 FROM wp_postmeta t1 
+				INNER JOIN wp_postmeta t2  
```

Comment:
Can we extract the where condition here and in `detect()` to a function so that it uses the same? That way we're guaranteed that `remove()` will only ever remove records that were returned in `detect()` even if we would make changes to the where condition.

Reply:
If we use the condition that is used currently for the delete in the detect as well, then we can easily make it show not only the posts, but also which meta keys are duplicated and a count per post_id + meta_key combination. That could help us in quickly seeing which part of the code is causing the duplication.